src/components/policy/policy_regular/test/policy_manager_impl_test.cc

```diff
@@ -563,7 +563,7 @@ TEST_F(PolicyManagerImplTest2,
   manager->CheckPermissions(
       app_id1, std::string("FULL"), "Alert", input_params, output);
   // Check RPC is disallowed
-  EXPECT_EQ(::policy::kRpcAllowed, output.hmi_level_permitted);
+  EXPECT_EQ(::policy::kRpcDisallowed, output.hmi_level_permitted);
```

Comment:
@okozlovlux, test named CheckPermissions_SetRevokedAppID_ExpectRPCDisallowed. I suppose, it was mistake in test.